src/tweet.js

```diff
@@ -33,6 +33,8 @@ export default class Tweet {
           this.getTransition().start();
         })
         .catch((error) => {
+          console.log('error displaying tweets', error);
+          if (this.logger) {this.logger.error(`Invalid Tweets`);}
```

Comment:
@jungeunyoon I think you don't need to check logger anymore here right based on your changes to the logger that checks if LMS is connected. Cheers

Reply:
@rodrigopavezi Isn't the purpose of checking if logger exists for preview mode where logger is null? Referencing a function `(error)` of an undefined class is problematic.

Reply:
@jungeunyoon I see, sorry.

Reply:
@rodrigopavezi no worried! good to point out anything you see 👍